freqtrade/main.py

```diff
@@ -6,12 +6,7 @@
 
 import logging
 import sys
-from typing import Any, Optional
-
-
-# check min. python version
-if sys.version_info < (3, 10):  # pragma: no cover
-    sys.exit("Freqtrade requires Python version >= 3.10")
```

Comment:
I wouldn't remove that.
While it's true that you can't install freqtrade on python < 3.10 - that doesn't mean you can't (attempt) to run it without installing (you can run freqtrade as module - where the pyproject.toml level "prevention" won't apply as far as i know)

While that still won't work - you'll get a very nasty "unsupported syntax" error - this ensures the user gets a proper error message. 

``` bash
pip install -r requirements.txt
python -m freqtrade -v
```